apps/web/src/components/Shared/Login/New/index.tsx

```diff
@@ -116,19 +157,18 @@ const NewProfile: FC<NewProfileProps> = ({ isModal = false }) => {
               <img className="h-60 w-60 rounded-lg" height={240} width={240} src={avatar} alt={avatar} />
             </div>
           )}
-          {/* TODO: reactivate avatar upload once we have a S3 bucket
           <div>
             <div className="flex items-center space-x-3">
               <ChooseFile onChange={(evt: ChangeEvent<HTMLInputElement>) => handleUpload(evt)} />
               {uploading && <Spinner size="sm" />}
             </div>
-          </div>*/}
+          </div>
         </div>
       </div>
       <Button
         className="ml-auto"
         type="submit"
-        disabled={!write || isLoading || !isConnected}
+        disabled={!write || loading || isLoading || !isConnected}
```

Comment:
Following one of my above message, can't we do this?
```suggestion
        disabled={!write || isCreationLoading || !isConnected}
```

Reply:
It's not exactly the same.
`loading` is `true` before we receive the "RelayerResult" data, so in terms of UX is disables the button a tiny bit earlier.